network-api/networkapi/wagtailpages/tests/customblocks/test_blog_cta_card_block.py

```diff
@@ -0,0 +1,34 @@
+from django.test import TestCase
+
+from networkapi.wagtailpages.factory import customblocks as customblock_factories
+
+
+class TestBlogCTACardBlock(TestCase):
+    def test_blog_cta_card_all_optionals(self):
+        full_card = customblock_factories.BlogCTACardBlockFactory()
+        print(full_card)
```

Comment:
Hey @jhonatan-lopes just checking if we want to print `full_card` during this test or if maybe this was left here by accident

Reply:
Great catch, thanks! I have a tendency to leave those pesky prints behind :see_no_evil: